Dangerfile.df.kts

```diff
@@ -46,16 +46,28 @@ danger(args) {
         )
         val composePackageFilesCreated = createdFiles.any { it.startsWith("backpack-compose/src/main/kotlin") }
         if (composePackageFilesCreated && !composeUsageDetector && !isTrivial) {
-            warn("One or more package files were created, but `BpkComposeComponentUsageDetector.kt` wasn't updated.")
+            warn("One or more package files were created, but `BpkComposeComponentUsageDetector.kt` wasn't updated. If your component is an equivalent of a core component please add it to the detector.")
         }
 
         // If any screenshots were created, README should"ve updated.
         val screenshotsCreated = createdFiles.any { it.startsWith("docs") && it.contains("/screenshots/") }
-        if (screenshotsCreated) {
-            val readmeUpdated = allSourceFiles.any { it.startsWith("docs") && it.endsWith("README.md") }
-            if (!readmeUpdated) {
-                warn("One or more screenshot created, but `README.md` wasn't updated. Please include screenshots in README")
-            }
+        val readmeUpdated = allSourceFiles.any { it.startsWith("docs") && it.endsWith("README.md") }
+        if (screenshotsCreated && !readmeUpdated) {
+            warn("One or more screenshot created, but `README.md` wasn't updated. Please include screenshots in README")
+        }
+
+        // If a component was updated, the corresponding README, tests and screenshots should have been updated.
```

Comment:
added a few more checks - these might be a little noisy in some areas where no visual changes happen, but will be helpful in most cases where these changes are missed